### PR TITLE
Dynamic channels 1

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/utils/__init__.py [hunk 1]
```diff
@@ -510,4 +510,4 @@ def get_anonymous_user():
     Returns the anonymous user, originally created by django-guardian
     """
     from django.contrib.auth.models import User
-    return User.objects.get(username=settings.ANONYMOUS_USER_NAME)
```
Comment (rowanseymour) [inline temba/utils/__init__.py]: What's the reason for this change? Seems unrelated ?

File: templates/channels/channel_claim.haml [hunk 2]
```diff
@@ -447,42 +447,6 @@
...
-                %p.claim-text
-                  -trans "Blackmyna Number"
-
-          .span9
-            .claim-description
-              %div
-                -blocktrans
-                  Easily add a two way number you have configured with <a href="http://blackmyna.com">Blackmyna</a> using their APIs.
-
-      .claim-row.smscentral-channel
```
Comment (rowanseymour) [inline templates/channels/channel_claim.haml]: smscentral isn't converted in this PR so shouldn't be removed here, right?

**Comment by norkans7** [comment]
@rowanseymour this has been updated
